ios/RNGoogleMobileAds/RNGoogleMobileAdsAppOpenModule.h

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#import <Foundation/Foundation.h>
+
+#ifdef RCT_NEW_ARCH_ENABLED
+
+#import <RNGoogleMobileAdsSpec/RNGoogleMobileAdsSpec.h>
```

Comment:
This port will be a regression for macCatalyst builds - all of the google ads stuff has to be ifdef'd out there so that it at least compiles in that mode. It won't *do* anything but it will at least compile

see https://github.com/invertase/react-native-google-mobile-ads/pull/389/files

adding back in the big / bulk ifdefs there is needed to preserve it